docs/admin-guide/load-config.md

```diff
@@ -1,125 +1,92 @@
-# The load config
+# The Load Config File
```

Comment:
👍 Perhaps change the index (side column, see pic) accordingly?

![image](https://user-images.githubusercontent.com/28093618/116504450-95773b00-a8b8-11eb-9b37-6db6ae22408c.png)

Reply:
Fixed!